<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnUserData">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbConnectionStatus">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>-1</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connection Status : Disconnected</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDateTime">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>0</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time:_______            Date:________</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPeople">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../picture/image_tools/chamhoi.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnection">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbBorder">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>821</width>
     <height>9</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbConnectImage">
   <property name="geometry">
    <rect>
     <x>598</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../picture/image_tools/Disconnect.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxConnection">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>821</width>
     <height>801</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="cbxBaudRate">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxScanPort">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lbBaudRate">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Baud Rate:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxUserData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>801</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBoxDetailData">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>741</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Detailed Data</string>
     </property>
     <widget class="QLabel" name="lbName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name            :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDisplayName">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Waiting...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDisplayAddress">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Waiting...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbAddress">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Address        :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDisplayCity">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Waiting...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbCity">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>City               :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDisplayCountry">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Waiting...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbCountry">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Country        :</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioUsingFace">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Using face</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFaceRecognition">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Recogniton</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxImg">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>401</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Image and ID</string>
     </property>
     <widget class="QLabel" name="lbIDUserData">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID :_______________________</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbViewUser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>271</width>
        <height>301</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../picture/image_tools/user.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="radioUsingCard">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Using card</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxImageID">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>801</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="lbNameRegister">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbAddressRegister">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textCity">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbCityRegister">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>City</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textCountry">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbCountryRegister">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxRecordViewRecord">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Record View</string>
      </property>
      <widget class="QLabel" name="lbSearch">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Search Here:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textSearchHere">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioSearchName">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>20</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search by name</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioSearchID">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>20</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search by ID</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>531</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lbViewRegister">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>70</y>
         <width>201</width>
         <height>301</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../picture/image_tools/user.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbMovie">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>221</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnClearDataInput">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxGetID_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>200</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Get ID</string>
      </property>
      <widget class="QPushButton" name="btnScanRegister">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Scan</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="lbID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID    _________</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnBrowseImage">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>220</y>
        <width>171</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../picture/image_tools/Click_to_browse.png</normaloff>../picture/image_tools/Click_to_browse.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveDataUser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QTextEdit" name="textAddress">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnScanPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scan Port</string>
    </property>
   </widget>
   <zorder>btnScanPort</zorder>
   <zorder>btnConnect</zorder>
   <zorder>cbxBaudRate</zorder>
   <zorder>cbxScanPort</zorder>
   <zorder>lbBaudRate</zorder>
   <zorder>groupBoxUserData</zorder>
  </widget>
  <widget class="QPushButton" name="btnEditData">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registration / Edit User Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
